Assets/AppCenter/Editor/AndroidPostBuild.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using UnityEditor.Android;
+using UnityEngine;
+
+namespace Assets.AppCenter.Editor
+{
+    class AndroidPostBuild : IPostGenerateGradleAndroidProject
+    {
+        private const string GOOGLE_SERVICES_VERSION = "4.0.1";
+        private const string FIREBASE_CORE_VERSION = "16.0.1";
+        private const string FIREBASE_MESSAGING_VERSION = "17.0.0";
+
+        public int callbackOrder { get { return 0; } }
+
+        public void OnPostGenerateGradleAndroidProject(string path)
+        {
+            var settings = AppCenterSettingsContext.SettingsInstance;
+            if (settings.UsePush)
+            {
+                MoveGoogleJsonFile(path);
+                SwapGoogleAndJcenter(path);
```

Comment:
Do we need to proceed w/ `SwapGoogleAndJcenter` and further if e.g. `google-services` was not found?

Reply:
Yeah probavly not.